packages/cosmos_auth/pubspec.lock

```diff
@@ -244,6 +244,13 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "0.12.11"
+  material_color_utilities:
+    dependency: transitive
+    description:
+      name: material_color_utilities
+      url: "https://pub.dartlang.org"
+    source: hosted
+    version: "0.1.3"
```

Comment:
Why do we need to add `material_color_utilities` inside `cosmos_auth` package?

Reply:
It was added by the new flutter version automatically